### ISSUE TITLE
Hex logo

### ISSUE THREAD

**Issue by sbfnk** [epiforecasts/EpiNow2#791]
Do we want one? Here's an initial attempt.

<img width="229" alt="image" src="https://github.com/user-attachments/assets/5085175c-bb76-46b6-8509-cd5b9a04c9bb">

**Comment by seabbs** [comment]
Yes I think we do. Could maybe make the same sort of thing but horizontal?

**Comment by athowes** [comment]
> e.g.?

I think this one is good. If possible it could be nice to have some elements of stylistic similiarity between all Epiforecasts / epinowcast packages.

**Comment by jamesmbaazam** [comment]
Is this an svg? I could play around with some options.

**Comment by jamesmbaazam** [comment]
I feel like the white space below the plot is a bit awkward so the plot should probably sit on the bottom line.

**Comment by sbfnk** [comment]
SVG versions at:

https://u.pcloud.link/publink/show?code=XZpcPI0Z49hkS15tubLOWqpHYvjHUh1nEu07
https://u.pcloud.link/publink/show?code=XZ8cPI0ZCsnuPnSgDW8IjxTHkzr0qmEiHbt7

Alternative would be to create it from a script as e.g. in https://github.com/epinowcast/epidist/blob/main/inst/make_hexsticker.R which would probably help with the stylistic continuity that @athowes has suggested.

**Comment by seabbs** [comment]
I think I prefer the results we get from `hexsticker` but I don't feel strongly at all

**Comment by seabbs** [comment]
See https://github.com/epinowcast/epidist/issues/363 for a very critical whitespace issue this might run into and a solution from @athowes.

**Comment by athowes** [comment]
> See https://github.com/epinowcast/epidist/issues/363 for a very critical whitespace issue this might run into and a solution from @athowes.

Btw this works best if you increase the `dpi` up to 1200 then use the solution (if lower it trims it in an ugly way).

**Comment by jamesmbaazam** [comment]
What's the progress on this?

**Comment by athowes** [comment]
I am in favour of the mak a script similar to https://github.com/epinowcast/epidist/blob/main/inst/make_hexsticker.R solution.

**Comment by sbfnk** [comment]
I agree

**Comment by seabbs** [comment]
To return to content I like the idea of showing the reconstructed cases by infection date as that is the USP of the package vs alternatives and something that is important to communicate (even though say Rt might be tempting)

**Comment by seabbs** [comment]
@athowes any chance we can bribe you with a paper authorship to have a crack at this : )

**Comment by athowes** [comment]
Haha deal, will aim for doing it an evening this week / the weekend

**Comment by jamesmbaazam** [comment]
@athowes I'm going to take a stab at this and we can work off it if that's alright with you.

**Comment by athowes** [comment]
Oops! Sorry to let this slip. Happy for you to try or happy to have another go to do it this week.
